pom.xml

```diff
@@ -50,6 +50,13 @@
 
     <dependencies>
         <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.0</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
`<scope>compile</scope>` is the default; you don't need to say it.

This is fine for now, but I'd hold off a bit before finalizing this and look into Jackson, because Jersey has a Jackson integration package.